app/src/main/java/net/skyscanner/backpack/demo/compose/AppSearchModalStory.kt

```diff
@@ -119,15 +117,6 @@ internal fun DefaultAppSearchModalSample(
             onClose = { showModal.value = false },
             onInputChanged = { destination.value = it },
             clearAction = BpkClearAction(stringResource(id = R.string.text_field_clear_action_description)) { destination.value = "" },
-            behaviouralCallback = object : BehaviouralCallback {
```

Comment:
it's hard to provide a sample here as it requires the handling of the `BpkClickHandleScope` (as that needs to be passed to the content)